packages/github/src/context.ts

```diff
@@ -7,7 +7,7 @@ export class Context {
   /**
    * Webhook payload object that triggered the workflow
    */
-  payload: WebhookPayload
+  payload?: WebhookPayload
```

Comment:
Before, we returned an object with every field optional. If we can't actually create the object, we should just return undefined, otherwise we allow users to cast to what they need.